pkgs/version/version.go

```diff
@@ -0,0 +1,46 @@
+package version
```

Comment:
I guess the conventions is `pkg`, not `pkgs`

Reply:
this may have multiple packages, upcoming packages are `telemetry` package and `segment` package which may rest in this. I feel folder name `pkgs` is more appropriate

Reply:
Ack. As it's a recognised convention.